webpack.config.js

```diff
@@ -5,6 +5,9 @@ const fs = require('fs');
 const path = require('path');
 
 module.exports = (env, argv) => { // eslint-disable-line no-undef
+    env = env ? env : {};
+    argv = argv ? argv : {};
```

Comment:
these fallbacks are already present on the `2.3` branch:
https://github.com/sulu/sulu/blob/100091add44ce79fa37454abf1db58ee955537cf/webpack.config.js#L7-L9